src/DacpacTool/PackageBuilder.cs

```diff
@@ -391,9 +391,12 @@ private List<int> ParseSuppressionList(string suppressionList)
 
         public void GenerateCreateScript(FileInfo dacpacFile, string databaseName)
         {
-            databaseName ??= Path.GetFileNameWithoutExtension(dacpacFile.Name);
-            var scriptFileName = $"{databaseName}_Create.sql";
+            if (databaseName is null)
```

Comment:
We should probably use string.IsNullOrWhiteSpace here?

Reply:
Good catch